application/models/Logbook_model.php

```diff
@@ -4222,6 +4222,14 @@ function import($record, $station_id = "0", $skipDuplicate = false, $markClublog
 				$qslmsg = $station_qslmsg;
 			}
 
+			// Only import SIG_INFO and SIG_INFO_INTL if SIG is set. Discard otherwise as we do not now which activity group the reference belongs to
+			if (!isset($record['sig']) || $record['sig'] == '') {
```

Comment:
maybe bit cleaner and easier to read :)

```suggestion
			if (empty($record['sig'])) {
```